shimmingtoolbox/optimizer/lsq_optimizer.py

```diff
@@ -3,43 +3,38 @@
 
 import numpy as np
 import scipy.optimize as opt
-from typing import List
 import warnings
+from typing import List
```

Comment:
This should be in alphabetical order

Reply:
So I should keep typing in the same place as before ?